pkg/operator/observe_config.go

```diff
@@ -56,16 +56,28 @@ func NewConfigObserver(
 	return c
 }
 
-// sync reacts to a change in prereqs by finding information that is required to match another value in the cluster. This
-// must be information that is logically "owned" by another component.
-// TODO(juanvallejo): update this to actually watch this operator's config
+// sync reacts to a change in controller manager images.
 func (c ConfigObserver) sync() error {
 	operatorConfig, err := c.operatorConfigClient.Get("instance", metav1.GetOptions{})
 	if err != nil {
 		return err
 	}
 	observedConfig := map[string]interface{}{}
 
+	controllerManagerImages, err := c.kubeClient.CoreV1().ConfigMaps(operatorNamespaceName).Get("openshift-controller-manager-images", metav1.GetOptions{})
```

Comment:
structure it with the observeConfig function describe here: https://github.com/openshift/cluster-kube-apiserver-operator/pull/74 with the additional `rest.Config` being passed.